include/mbedtls/md.h

```diff
@@ -227,13 +228,13 @@ mbedtls_md_type_t mbedtls_md_get_type( const mbedtls_md_info_t *md_info );
  * \param md_info   The information structure of the message-digest algorithm 
  *                  to use.
  *
- * \return          The name of the message-digest output.
+ * \return          The name of the message digest.
  */
 const char *mbedtls_md_get_name( const mbedtls_md_info_t *md_info );
 
 /**
- * \brief           This function prepares the message-digest context for  
- *                  a new message. 
+ * \brief           This function Extracts the message-digest name from the  
```

Comment:
Is it intentional that `Extracts` is in capitals?

Reply:
Fixed